ViAn/GUI/mainwindow.cpp

```diff
@@ -364,6 +375,7 @@ void MainWindow::init_view_menu() {
     view_menu->addAction(toggle_bookmark_wgt);
     view_menu->addAction(toggle_queue_wgt);
     view_menu->addAction(toggle_ana_settings_wgt);
+    view_menu->addAction(toggle_zoom_preview_wgt);
```

Comment:
Maybe a small thing but this line should be moved up two steps so the zoom preview is after the bookmark widget. By looking at the other changes that's where it should be. It also makes the context menu and view (window) menu the identical.